samples/ar-model-viewer/src/main/java/io/github/sceneview/sample/armodelviewer/MainFragment.kt

```diff
@@ -61,9 +61,10 @@ class MainFragment : Fragment(R.layout.fragment_main) {
             }
             editableTransforms = EditableTransform.ALL
         }
-        sceneView.addChild(modelNode)
-        // Quick workaround until the Node Pick is fixed
-        sceneView.gestureDetector.onTouchNode(modelNode)
+        sceneView.apply {
+            addChild(modelNode)
+            gestureDetector.selectedNode = modelNode
```

Comment:
@grassydragon If the node picking works well, we could remove this workaround. Otherwise we should add a comment that this line makes the model start as selected or something. I might be confusing for people looking at the sample why there's this line and if it's mandatory.